src/components/CreateManifestForm/CreateManifestForm.tsx

```diff
@@ -27,23 +24,28 @@ interface CreateManifestFormProps {
 }
 
 const CreateManifestForm: FC<CreateManifestFormProps> = (props) => {
-  const { satelliteVersions, handleModalToggle, submitForm, isLoading, isError, isSuccess } = props;
-  const {
-    control,
-    handleSubmit,
-    formState: { errors }
-  } = useForm({ mode: 'onBlur' });
+  const { isError: hasSatelliteVersionsError } = useSatelliteVersions();
+  const { data: createManifestResponseData, isError: errorCreatingManifest } =
+    useCreateSatelliteManifest();
+  type Validate = 'default' | 'error' | 'success';
+  const { satelliteVersions, handleModalToggle, isLoading, submitForm, isError, isSuccess } = props;
+  const [inputFieldBlur, setinputFieldBlur] = React.useState(false);
+  const [dropdownFieldBlur, setDropdownFieldBlur] = React.useState(false);
   const [manifestName, setManifestName] = useState('');
+  const [manifestType, setManifestType] = useState('');
   const { addSuccessNotification, addErrorNotification } = useNotifications();
+  const [invalidTypeText, setInvalidTypeText] = React.useState('');
+  const [nameValidated, setNameValidated] = React.useState<Validate>('default');
+  const [typeValidated, setTypeValidated] = React.useState<Validate>('default');
+  const [nameFieldHelperText, setNameFieldHelperText] = React.useState(
+    'Your manifest name must be unique and must contain only numbers, letters, underscores, and hyphens.'
+  );
+  const [invalidNameFieldText, setInvalidNameFieldText] = React.useState(
+    'Name requirements have not been met.Your manifest name must be unique and must contain only numbers, letters, underscores, and hyphens.'
+  );
```

Comment:
I don't think these even need to be state - just constant variables.

The formgroup I think should handle when to show which - you just need to tell it which string is valid and which is invalid which you are already doing. 

```suggestion
  const nameFieldHelperText = 'Your manifest name must be unique and must contain only numbers, letters, underscores, and hyphens.';
  const invalidNameFieldText = `Name requirements have not been met. ${nameFieldHelperText}`
```